documentcloud/entities/permissions.py

```diff
@@ -0,0 +1,16 @@
+from rest_framework import permissions
+
+
+class IsOwnerOrReadOnly(permissions.BasePermission):
```

Comment:
I would avoid making a custom permission class for this.  There is a rules package that I use that makes defining permissions a little easier.  See the rules.py in the Documents app for an example.  We can go over this in more detail at some point also.